microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -169,8 +169,7 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 
 <!-- Above, should pick one way. Check UI for privacy updates -->
 
-You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
-Collections aren't available when using **InPrivate** or **Guest** modes.
+If you're signed in with a Microsoft account, your collections, notes, comments, and images are automatically saved on Microsoft cloud for free.  If you want to turn off this service, disable Collections for this device as follows: go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Show Collections and follow content creators in Microsoft Edge**.  Collections aren't available when using Guest mode or InPrivate browsing for your Microsoft account.
```

Comment:
Add a paragraph about the behavior for when you're signed in with a work or school account.
@yuanjunz

Reply:
When you sign in with a work or school account, your collections are automatically synced across all signed-in Microsoft Edge devices when sync is turned on. If you want to stop syncing your Collections, you can turn off sync for Collections from `edge://settings/profiles/sync`, then all Collections data is stored locally with your Microsoft Edge profile on the device.

Reply:
added